kubernetes/aks/system/monitoring/helm/kube-prometheus-stack-values.yaml

```diff
@@ -940,7 +940,7 @@ alertmanager:
 ## Using default values from https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml
 ##
 grafana:
-  enabled: true
+  enabled: false
```

Comment:
For my understanding: is this getting disabled because this is the built-in version and we provide the service separetely?